version/version.go

```diff
@@ -21,7 +21,7 @@ import "github.com/astaxie/beego"
 var (
 	// no need to modify
 	// please use:
-	// 	go build -ldflags "-X github.com/ServiceComb/service-center/version.VERSION=x.x.x"
+	// 	go build -ldflags "-X github.com/apache/incubator-servicecomb-service-center/version.VERSION=x.x.x"
 	// to set these values.
 	VERSION   = "0.0.1"
```

Comment:
Do we need to update the version to reflect the current release version.

Reply:
No, the version number can be specified while making a new build, the version number is taken from env variable using go ldlflags, for example
```
export GIT_COMMIT=XXXX
export BUILD_NUMBER=XXXX
GO_LDFLAGS="${GO_LDFLAGS} -X 'github.com/apache/incubator-servicecomb-service-center/version.BUILD_TAG=$(date +%Y%m%d%H%M%S).$BUILD_NUMBER.$GIT_COMMIT'"
GO_LDFLAGS="${GO_LDFLAGS} -X 'github.com/apache/incubator-servicecomb-service-center/version.VERSION=XXXXX'"
go build --ldflags "${GO_LDFLAGS}"
```